entry/src/poh.rs

```diff
@@ -92,7 +94,7 @@ impl Poh {
 
         // If the hashes_per_tick is variable (std::u64::MAX) then always generate a tick.
         // Otherwise only tick if there are no remaining hashes
-        if self.hashes_per_tick < std::u64::MAX && self.remaining_hashes != 0 {
+        if self.hashes_per_tick != VARIABLE_HASHES_PER_TICK && self.remaining_hashes != 0 {
```

Comment:
These are semantically equivalent because there is no u64 larger than std::u64::MAX